third_party/move/move-compiler-v2/src/plan_builder.rs

```diff
@@ -0,0 +1,700 @@
+// Copyright (c) The Diem Core Contributors
```

Comment:
It seems that some code is from the [plan_builder](https://github.com/aptos-labs/aptos-core/blob/main/third_party/move/move-compiler/src/unit_test/plan_builder.rs) for V1. Is it possible to reuse of the code there?

Reply:
It's my turn to be like Wolfgang.  Read the PR.  I started with the V1 plan_builder and modify it to use the V2 data sources instead of V1.  One commit just copies `plan_builder.rs` to `move-compiler-v2/src`, then the next commit modifies it and gets it to work with V2.  The code parallels the code there, but there is very little re-use (except perhaps string constants) because almost every data item becomes a new type (even when the type names look the same).